spec/views/advertisements/create.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "advertisements/create.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
If you aren't going to implement these specs, you should delete them.

Personally I'm not a big fan of view specs.